### PR TITLE
Tag DDS IDs telemetry as PackageData

### LINKED ISSUE #7843: We log DDS IDs into telemetry without proper attribution
This does not look right - this.id is user provided ID (at least today).
It should be marked with TelemetryDataTag.PackageData.

Can you please inspect code overall (not just this case) for similar issues?
Can we catch somehow automatically issues like that?

```typescript
                // TODO: dataStoreId may require a different tag from PackageData #7488
                throw new DataCorruptionError("channelFactoryNotRegisteredForAttachMessageType", {
                    channelId: this.id,
                    dataStoreId: {
                        value: this.dataStoreContext.id,
                        tag: TelemetryDataTag.PackageData,
                    },
                    dataStorePackagePath: this.dataStoreContext.packagePath.join("/"),
                    channelFactoryType: this.attachMessageType,
                });
```

### PR DESCRIPTION
**Pull request by kian-thompson**
#7843 
Note: These were the only places I found we were logging IDs in telemetry. Everywhere else was properly removing them using `extractSafePropertiesFromMessage(message: ISequencedDocumentMessage)` or was already tagging as `PackageData`.

### CODE REVIEW

**Comment by kian-thompson** [comment]
Not sure, but the following might also need to be tagged:
image

**Comment by vladsud** [comment]
> Not sure, but the following might also need to be tagged: image

Yep, it should be updated as well.

**Review by vladsud — APPROVED** [review]

File: packages/runtime/container-runtime/src/summaryGenerator.ts [hunk 1]
```diff
@@ -389,7 +389,10 @@ export class SummaryGenerator {
                 duration: broadcastDuration,
                 referenceSequenceNumber: summarizeOp.referenceSequenceNumber,
                 summarySequenceNumber: summarizeOp.sequenceNumber,
-                handle: summarizeOp.contents.handle,
+                handle: {
+                    value: summarizeOp.contents.handle,
```
Comment (kian-thompson) [inline packages/runtime/container-runtime/src/summaryGenerator.ts:393]: This one seemed to potentially be sensitive data as well

Comment (vladsud) [reply packages/runtime/container-runtime/src/summaryGenerator.ts:393]: handles are produces by storage, so they cannot be coming from code (or user).
They are either random numbers or checksums (like sha1), so it's safe to have them untagged.

**Review by vladsud — APPROVED** [review]

**Pull request merged**